rubocop.gemspec

```diff
@@ -1,7 +1,6 @@
 # frozen_string_literal: true
 
-$LOAD_PATH.unshift File.expand_path('lib', __dir__)
-require 'rubocop/version'
+require_relative 'lib/rubocop/version'
 require 'English'
```

Comment:
I think now you can remove this, as it was needed for `$RS` that was removed.

Reply:
Good catch! Another nice side effect of the change.